CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 1.2.0
+
+## Features and improvements
+* Get start-docs working on linux ([#8486](https://github.com/mapbox/mapbox-gl-js/pull/8486))
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Make requirements for text offset props more precise ([#8418](https://github.com/mapbox/mapbox-gl-js/pull/8418))
+* Get Codegen to remove extraneous newlines. ([6fdbdeb](https://github.com/mapbox/mapbox-gl-js/commit/6fdbdebd25de292ab1d42e4e6beff05fbf696d2e))
```

Comment:
This shouldn't be in changelog (no user-facing changes) and this is addressing an earlier PR in the same release.